runtime/util/moninfo.c

```diff
@@ -64,15 +64,15 @@ getObjectMonitorOwner(J9JavaVM *vm, J9VMThread *vmThread, j9object_t object, UDA
 
 	Trc_VMUtil_getObjectMonitorOwner_Entry(vm, object, pcount);
 
-	if (!LN_HAS_LOCKWORD(vmThread,object)) {
+	if (!LN_HAS_LOCKWORD_VM(vm,object)) {
```

Comment:
formatting: space after comma

Reply:
I did not change any existing formatting.

Reply:
Since you're changing this line, please fix it.

Reply:
Done.